website/docs/docs/cloud/dbt-cloud-ide/ide-glossary.md

```diff
@@ -0,0 +1,261 @@
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-markdown-with-save.jpg" width="100%" title="The Markdown Preview subtab renders markdown code below the Editor tab."/>
+
+#### CSV Preview subtab
+The CSV Preview subtab renders csv code in a .csv file in your seed directory into tabular format. This is updated as you make edits to your code! 
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-csv.jpg" width="100%" title="View csv code in the CSV Preview subtab below the Editor tab."/>
+
+## Invocation history
+
+The Invocation History Drawer contains all information related to dbt invocations in the IDE. This menu can be accessed by the `^` icon next to the Command Bar, or automatically by invoking a command. 
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/invocation-components-with-save.jpg" width="100%" title="The Command History returns a log and detail of all your dbt Cloud invocations."/>
+
+A &mdash; [Invocation History list](#invocation-history-list)<br />
+B &mdash; [Invocation Summary](#invocation-summary)<br />
+C &mdash; [System Logs toggle](#system-logs-toggle)<br />
+D &mdash; [Result Status tabs](#result-status-tabs)<br />
+E &mdash; [Node result](#node-result)<br />
+F &mdash; [Command Control button](#command-control-button)<br />
+
+### Invocation History list
+
+The Invocation History List in the left hand panel of the Invocation History Drawer shows summary details about previous invocations in the IDE. This includes the command, the branch name, the status of the command, and the total elapsed time. 
+
+### Invocation Summary
+
+The Invocation Summary shows the summary details of the selected invocation from the Command History List. This includes the command invoked, the status of the selected command (`Running` if the command is still in progress), the active git branch at the time of the invocation, and the timestamp of the invocation.
+
+### System Logs toggle
+The System Logs Toggle allows the user to see the full stdout and debug logs for entirety of the invoked command. 
+
+### Result Status tabs
+The Results Status Tabs group executed nodes from the command by their result. Clicking on these will filter the Node Status List by the corresponding status. Statuses include:
+
+- Pass - successful invocation of a node
+- Warn (tests only) - test executed with warning
+- Error - database error when executing node or failure in test
+- Skip - nodes that did not run due to upstream error
+- Queued - nodes that have yet to execute
+
+### Node result
+Each node executed during a dbt command invocation will be summarized in a Node Result toggle, that contains the summary (stdout) and debug logs to easily see the details about each node. The Node Results List contains an entry for every invoked node. 
+
+### Node result list
+
+The Node result list contains the full list of each Node Result invoked during the dbt run. This can be filtered by clicking on any of the Result Status tabs. 
+### Command Control button
```

Comment:
general casing for all these headings is inconsistent... are we doing Title Case or Sentence case?

Reply:
It should be sentence case per our style guide @bossnunta, however UI elements override that. So if the UI name is title case, then the docs need to reflect that.